### ISSUE TITLE
[Accessibility] Landmarks should have a unique role or role/label/title (i.e. accessible name) combination

### ISSUE THREAD

**Issue by sparklerfish** [kaltura/playkit-js-ui#1155]
Axe devtools reports the following error for the `playkit-video-player` div with `role="region"`:
"Landmarks should have a unique role or role/label/title (i.e. accessible name) combination"

It looks like this was the result of an erroneous attempted fix to an already-fixed problem:
1. An aria-label is added to the div (#1075).  This results in the Axe error that elements must have valid ARIA attributes.
2. `role="region"` is added to the div (#1087).  This is correct and fixes the above error, and conforms to WCAG requirements for landmark regions to have an accessible name
3. The aria-label is _removed_ from the div (#1093).  Based on the PR description, it looks like this was attempting to fix the first issue, but it had already been fixed, so this instead caused a new accessibility issue by creating landmarks without names.

Simply reverting #1093 will resolve the issue and result in a div with valid attributes.
